<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Devil-Machine" name="User Details Test" tests="1" failures="1" timestamp="6 Feb 2022 16:47:04 GMT" time="5.803" errors="0">
  <testcase name="BookingCompletionTest" time="2.401" classname="testCases.UserDetailsPageTest">
    <failure type="java.lang.AssertionError" message="Unable to navigate to Booking page expected [Booking Confirm] but found [Page not found at /rent/75]">
      <![CDATA[java.lang.AssertionError: Unable to navigate to Booking page expected [Booking Confirm] but found [Page not found at /rent/75]
	at testCases.UserDetailsPageTest.BookingCompletionTest(UserDetailsPageTest.java:27)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- BookingCompletionTest -->
</testsuite> <!-- User Details Test -->
